core/src/test/resources/org/elasticsearch/index/analysis/test1.json

```diff
@@ -17,10 +17,6 @@
                 },
                 "my":{
                     "type":"myfilter"
-                },
```

Comment:
Oh! I see. You don't need to get rid of it because you can plugin mocks into it. Great.